main.js

```diff
@@ -1,6 +1,62 @@
-import './style.css'
+const APIKEY="api_key=f4fa6a2d7a3bb9c2777369817722c765" ;
+const BASE_URL="https://api.themoviedb.org/3";
+const API_URL=`${BASE_URL}/discover/movie?sort_by=popularity.desc&${APIKEY}`;
+const IMG="https://image.tmdb.org/t/p/w1280";
+const SEARCH =`${BASE_URL}/search/movie?&${APIKEY}`;
 
-document.querySelector('#app').innerHTML = `
-  <h1>Hello Vite!</h1>
-  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
-`
+const noresult=document.querySelector('.noresult');
+const main =document.querySelector('.main');
+const movielist= document.querySelector('.movielist');
+const form= document.querySelector(".form");
+const search= document.querySelector(".search");
```

Comment:
Be consistent with quote annotations. Use either single quotes everywhere or double quotes everywhere. Don't mix them both.